example/src/stories/RadioButton.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import type {ComponentMeta, ComponentStory} from '@storybook/react'
+
+import {RadioButton} from 'rn-base-component'
+
+export default {
+  title: 'components/RadioButton',
+  component: RadioButton,
+} as ComponentMeta<typeof RadioButton>
+
+export const Basic: ComponentStory<typeof RadioButton> = args => <RadioButton {...args} />
```

Comment:
If we can make it in center with a margin, It will be more beauty

Reply:
could you explain more detail. I don't get your point